db/nodes_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/kardiachain/explorer-backend/types"
 )
 
-func GetMgo() (*mongoDB, error) {
+func GetTestMgo() (*mongoDB, error) {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported func GetTestMgo returns unexported type *github.com/kardiachain/explorer-backend/db.mongoDB, which can be annoying to use (golint)